src/foam/nanos/auth/UserAndGroupAuthService.java

```diff
@@ -277,6 +274,10 @@ public boolean check(foam.core.X x, String permission) {
     return checkPermission(x, new AuthPermission(permission));
   }
 
+  public boolean validatePassword( String newPassword) {
+    return Password.isValid(newPassword);
```

Comment:
I think that the validation should be here directly, not in Password.

Reply:
If we put the validation here, won't it still be the case that there will be two places where we define what appropriate validation is (Password and here)? 
I'm fine with putting the validation here, just trying to understand why its a better choice.
The Password  model seems like a more intuitive place to put the rule, and we can always still overwrite this function to put custom rules in place.